extensions/azurePublish/src/components/ChooseProvisionAction.tsx

```diff
@@ -139,28 +139,28 @@ const ImportActionContent = () => {
       </Summary>
       <Details>
         <ResourceTitle>
-          <Text>{formatMessage('Microsoft Application Registration')}</Text>
+          <Text>{formatMessage('- Microsoft Application Registration')}</Text>
```

Comment:
Does it make more sense to have the '-' outside of the `formatMessage` function call?
```suggestion
          <Text>- {formatMessage('Microsoft Application Registration')}</Text>
```

Reply:
Handled